### ISSUE TITLE
Swap menu theme module to use round theme module.

### ISSUE THREAD

**Issue by Lanred-Dev** [monke-mob/monke-activities#27]
The round branch and menu branch both have different theme modules. The round branch module is the most up-to-date of the two and includes better QOL improvements when compared to the menu branch one. The menu branch module should be swapped and updated to use the round branch module.

**Comment by Lanred-Dev** [comment]
Superseded by #30. However still relevant after #30 is completed.
